fabric-particles-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-particles-v1"
+version = getSubprojectVersion(project, "0.1.1")
+
+dependencies {
+	compile project(path: ':fabric-api-base', configuration: 'dev')
```

Comment:
This should probably also depend on `fabric-textures-v0`, since `ClientSpriteRegistryCallback` is required to register sprites to the particle atlas for use with particles - even though it's not used directly by the API, this would ensure that it's present for use.